.mocharc.cjs

```diff
@@ -8,33 +8,27 @@ const {
   packagesExpr,
   packagesList,
   testsDir,
-  testSuffixPerformance,
-  testSuffixTest,
+  testSuffix,
   testTypePerformance,
   testTypeIntegration,
   testTypeTests,
   testTypeUnit,
 } = values;
 
 let testTypeDir;
-let testTypeSuffix;
 switch (process.env.TEST_TYPE) {
 case testTypePerformance:
-  testTypeDir = "{.,benchmark,performance}";
-  testTypeSuffix = testSuffixPerformance;
+  testTypeDir = "performance";
```

Comment:
This string (`"performance"`) [is duplicated in the `get-filters.js` script](https://github.com/ericcornelissen/webmangler/pull/261/files#diff-2797ab6994f91c2ca2c603d5b1d9e833585ea9c42fd3b3504fa607dfb17aef18R51) and would ideally be exported from the `.values.cjs` file.

The same should also be applied to [`"integration"`](https://github.com/ericcornelissen/webmangler/pull/261/files#diff-36b1f8016764408096aeb19a2f9a43674fa49f94fd46fc0c9f8e8657967281d8L27), [`"{common,unit}"`](https://github.com/ericcornelissen/webmangler/pull/261/files#diff-36b1f8016764408096aeb19a2f9a43674fa49f94fd46fc0c9f8e8657967281d8L31) and [`"{.,common,integration,unit}"`](https://github.com/ericcornelissen/webmangler/pull/261/files#diff-36b1f8016764408096aeb19a2f9a43674fa49f94fd46fc0c9f8e8657967281d8L36) (preemptively).